utils/build-script

```diff
@@ -662,6 +670,20 @@ the number of parallel build jobs to use""",
             "--skip-test-watchos",
         ]
 
+    if not args.cc or not args.cxx:
+        host_clang = swift_build_support.clang.host_clang(
+            xcrun_toolchain=args.darwin_xcrun_toolchain)
+        if not host_clang:
+            print_with_argv0("Can't find clang. "
+                             "Please install clang-3.5 or a later version.")
+            return 1
+
+        if not args.cc:
+            args.cc = host_clang.cc
```

Comment:
Instead of modifying command line parameters, could you just re-use the `host_clang` variable?  You can create an instance with `CompilerExecutable(cc=theCC, cxx=theCXX)`.